src/commands/lambda/flare.ts

```diff
@@ -40,105 +49,170 @@ export class LambdaFlareCommand extends Command {
     this.context.stdout.write(flareRenderer.renderLambdaFlareHeader(this.isDryRun))
 
     // Validate function name
+    const errorMessages = []
     if (this.functionName === undefined) {
-      this.context.stderr.write(commonRenderer.renderError('No function name specified. [-f,--function]'))
-
-      return 1
+      errorMessages.push(commonRenderer.renderError('No function name specified. [-f,--function]'))
     }
 
     // Validate region
-    let errorFound = false
-    const region = getRegion(this.functionName) ?? this.region ?? process.env[AWS_DEFAULT_REGION_ENV_VAR]
+    const region = getRegion(this.functionName ?? '') ?? this.region ?? process.env[AWS_DEFAULT_REGION_ENV_VAR]
     if (region === undefined) {
-      this.context.stderr.write(commonRenderer.renderNoDefaultRegionSpecifiedError())
-      errorFound = true
+      errorMessages.push(commonRenderer.renderNoDefaultRegionSpecifiedError())
     }
 
     // Validate Datadog API key
     this.apiKey = process.env[CI_API_KEY_ENV_VAR] ?? process.env[API_KEY_ENV_VAR]
     if (this.apiKey === undefined) {
-      this.context.stderr.write(
+      errorMessages.push(
         commonRenderer.renderError(
           'No Datadog API key specified. Set an API key with the DATADOG_API_KEY environment variable.'
         )
       )
-      errorFound = true
     }
 
     // Validate case ID
     if (this.caseId === undefined) {
-      this.context.stderr.write(commonRenderer.renderError('No case ID specified. [-c,--case-id]'))
-      errorFound = true
+      errorMessages.push(commonRenderer.renderError('No case ID specified. [-c,--case-id]'))
     }
 
     // Validate email
     if (this.email === undefined) {
-      this.context.stderr.write(commonRenderer.renderError('No email specified. [-e,--email]'))
-      errorFound = true
+      errorMessages.push(commonRenderer.renderError('No email specified. [-e,--email]'))
     }
 
-    if (errorFound) {
+    // Exit if there are errors
+    if (errorMessages.length > 0) {
+      this.context.stderr.write(errorMessages.join('') + '\n')
+
       return 1
     }
+    assert(this.functionName !== undefined)
 
     // Get AWS credentials
     this.context.stdout.write('\n🔑 Getting AWS credentials...\n')
     try {
       this.credentials = await getAWSCredentials()
     } catch (err) {
-      this.context.stderr.write(commonRenderer.renderError(err))
+      this.context.stderr.write(commonRenderer.renderError(err.message))
```

Comment:
What's the reason behind this change?

Reply:
Sounds good. If you come up with a PR to refactor validation, it would be a good opportunity to type guard the errors, I noticed you didn't do it, which caused this errors to not show.